### ISSUE TITLE
Edit profile: Alert section fails non text contrast

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#48]
URL: https://hcdev.openplus.ca/en/user/203/edit
User Agent: Chrome, Firefox
Language: both

Issue Description: Alert section fails contrast for exclamation and close button as contrast is below 3:1 for non-text items.

1. Alert (exclamation) fails contrast. Without hover, contrast is 1.42:1 and with hover, contrast is 2.44:1. 
2. Close button fails contrast. Without hover, contrast is 1.6:1 and with hover, contrast is 3.18:1. 

Steps to reproduce: Enter different passwords in Password and Confirm password fields

Expected results:
1. Contrast should be 3:1 for non text elements and 4.5:1 for text. WET displays this at the beginning of the alert section.
2. Contrast should be 3:1 for non text elements and 4.5:1 for text. This section should not be a section that user can close. Close button should be just removed.

Please follow WET format https://wet-boew.github.io/v4.0-ci/demos/formvalid/formvalid-en.html

![image](https://user-images.githubusercontent.com/14280062/128736114-6fcd21f3-a9dd-43d2-99cd-3a070cfebea7.png)

This is WET example:
![image](https://user-images.githubusercontent.com/14280062/128735044-63ea2cc7-a6bd-4074-9b02-6606da546ab4.png)

**Comment by qanitah** [comment]
Items have been removed that had colour issue. It still does not follow WET layout though.
